pkgs/development/python-modules/gasp/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder,
+fetchgit, tkinter, pygame, python3, xvfb-run }:
```

```suggestion
fetchgit, tkinter, pygame, python, xvfb-run }:
```
python3 would mix python versions which must be prevented at all costs.